pkgs/applications/editors/formiko/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, wrapGAppsHook
+, gobject-introspection
+, gtk3
+, docutils
+, gtksourceview
+, gtkspell3
+, librsvg
+, pygobject3
+, webkitgtk
+}:
+
+buildPythonApplication rec {
+
+  pname = "formiko";
+  version = "1.4.3";
+
+  src = fetchFromGitHub {
+    owner = "ondratu";
+    repo = "formiko";
+    rev = version;
+    sha256 = "0n7w585gbrpn2xcd5n04hivrjarpr2wj260y2kpxpgh93vn52sdi";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    gobject-introspection
+    gtk3
+  ];
+  propagatedBuildInputs = [
+    docutils
+    gobject-introspection
+    gtk3
+    gtksourceview
+    gtkspell3
+    librsvg
+    pygobject3
+    webkitgtk
+  ];
+
+  # Needs a display
+  doCheck = false;
+
+  meta = with lib; {
+    description = "reStructuredText editor and live previewer";
+    homepage = "https://github.com/ondratu/formiko";
+    license = licenses.bsd2; # May be wrong
```

```suggestion
    license = licenses.bsd3;
```

https://metadata.ftp-master.debian.org/changelogs//main/f/formiko/formiko_1.3.0-1_copyright